fava/serialisation.py

```diff
@@ -9,8 +9,12 @@
 """
 
 import decimal
+import ast
 import functools
+import operator
 import re
+from io import StringIO
+from tokenize import generate_tokens, untokenize, NAME, NUMBER, OP, STRING
```

Comment:
I'd prefer to not depend on the undocumented generate_tokens function, could you change the implementation to use tokenize instead?